CASMINST-4219 - Add instructions to merge in new CSM 1.2 VCS content to upgrade procedure (#1086)
Cray-HPE/docs-csm: upgrade/1.2/Stage_3.md

```bash
ncn-m002# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/csm-upgrade.sh
```

> ## Update the CSM configuration management repository

Reviewer: ```suggestion
## Update the CSM Configuration Management Repository
```

> ## Update the CSM configuration management repository

Reviewer: ```suggestion
## Update the CSM Configuration Management Repository
```

1. Obtain the VCS login credentials.

   ```bash
   ncn-m002# VCS_USER=$(kubectl -n services get secret vcs-user-credentials -o jsonpath='{.data.vcs_username}' | base64 --decode)
   ncn-m002# VCS_PASSWORD=$(kubectl -n services get secret vcs-user-credentials -o jsonpath='{.data.vcs_password}' | base64 --decode)
   ```

1. Clone the CSM configuration management repository.

   ```bash
   ncn-m002# git clone https://${VCS_USER}:${VCS_PASSWORD}@api-gw-service-nmn.local/vcs/cray/csm-config-management.git
   ```

> 1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap

Reviewer: ```suggestion
1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap
```
```suggestion
1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap.
```

>    In the below example the import branch is `cray/csm/1.9.21`

Reviewer: ```suggestion
   In the following example, the import branch is `cray/csm/1.9.21`.
```

>    In the below example the import branch is `cray/csm/1.9.21`

Reviewer: ```suggestion
   In the following example, the import branch is `cray/csm/1.9.21`.
```

     recipes:
       cray-shasta-csm-sles15sp3-barebones.x86_64-csm-1.2:
         id: 9f6d9a43-148a-48f9-bec1-b3dbc4ec25d6
   ```

1. Ensure content is up to date.

   ```bash
   ncn-m002# cd csm-config-management && git checkout cray/csm/PRODUCT_VERSION && git pull
   Branch 'cray/csm/PRODUCT_VERSION' set up to track remote branch 'cray/csm/PRODUCT_VERSION' from 'origin'.
   Switched to a new branch 'cray/csm/PRODUCT_VERSION'
   Already up to date.
   ```

1. Create or update a working branch using the updated content installed during the upgrade.

   ```bash
   ncn-m002# git checkout -b integration && git merge cray/csm/PRODUCT_VERSION
   Switched to a new branch 'integration'
   Already up to date.
   ```

   Resolve any merge conflicts as necessary.

1. Push the changes to the repository.

   ```bash
   ncn-m002# git push --set-upstream origin integration
   ```

1. Capture most recent commit to use to update the CFS configuration and perform NCN Personalization.

   ```bash
   ncn-m002# git rev-parse --verify HEAD
   5616de1ffd375ac94a0f9c51f65dee3063b6be6e
   ```

1. Update the CFS configuration and perform NCN personalization.

>    Please follow the procedure from [Perform\_NCN\_Personalization.md](../../operations/CSM_product_management/Perform_NCN_Personalization.md) to update the CFS configuration and run NCN Personalization.

Reviewer: ```suggestion
   Refer to the [Perform NCN Personalization](../../operations/CSM_product_management/Perform_NCN_Personalization.md) procedure to update the CFS configuration and run NCN Personalization.
```

>    Please follow the procedure from [Perform\_NCN\_Personalization.md](../../operations/CSM_product_management/Perform_NCN_Personalization.md) to update the CFS configuration and run NCN Personalization.

Reviewer: ```suggestion
   Refer to the [Perform NCN Personalization](../../operations/CSM_product_management/Perform_NCN_Personalization.md) procedure to update the CFS configuration and run NCN Personalization.
```
